### ISSUE TITLE
preferred_versions.md out of date with what rush is actually doing

### ISSUE THREAD

**Issue by abdes** [microsoft/rushjs.io-website#71]
The documentation for preferred versions claims the following:
 > To control these effects Rush introduces a concept of “preferred versions”, which are dependencies that get explicitly added to the top-level common/temp/package.json.

That is not happening.

Additionally, the same concept is also the basis for the example given to configure git hooks. Well, the pretty-quick and prettier modules are not installed, and the node_modules under common/temp has no .bin directory.

## How to reproduce

```
$ git clone https://github.com/microsoft/rushstack
$ cd rushstack
$ rush update

(base)[~/Projects/rush-test/rushstack]$ ls -a common/temp/node_modules                                                                                                                           [master]
.             ..            .modules.yaml .pnpm

```
https://github.com/microsoft/rushstack/blob/3126f09a467c8ccef03b0c0422f350fc4a6a135d/common/config/rush/common-versions.json#L21

**Comment by octogonz** [comment]
@abdes This is because **rush.json** specifies the experimental `"useWorkspaces": true` which we are dogfooding in that repo.

**Comment by abdes** [comment]
> @abdes This is because **rush.json** specifies the experimental `"useWorkspaces": true` which we are dogfooding in that repo.

Yes, I am also using the same. Just wanted to make sure this issue is tracked somewhere. I will change the title to specify 'when pnpm useWorkspaces is true'
